Tests/text-formatting.lisp

```diff
@@ -0,0 +1,70 @@
+(cl:in-package #:clim-tests)
+
+(def-suite* :mcclim.text-formatting
+  :in :mcclim)
+
+(test normalize-margin-spec
+  (flet ((equals (spec-1 spec-2)
+           (and (= (length spec-1) (length spec-2))
+                (loop for (edge value) on spec-1 by #'cddr
+                      unless (equal value (getf spec-2 edge))
+                        do
+                           (return nil)
+                      finally
+                         (return t)))))
+    (is (equals (climi::normalize-margin-spec nil
+                                              '(:left (:relative 0)
+                                                :top (:relative 0)
+                                                :right (:relative 0)
+                                                :bottom (:relative 0)))
+                '(:left (:relative 0)
+                  :top (:relative 0)
+                  :right (:relative 0)
+                  :bottom (:relative 0))))
```

Comment:
Fiveam wants the expected value as the first argument. Putting the expected value first generates the report
```
(climi::normalize-margin-spec …)

  evaluated to

(:top …)
```

instead of

```
  (:top …)

evaluated to

  (:top …)
```

Reply:
done